apps/bluetooth/js/views/pair_expired_dialog.js

```diff
@@ -1,9 +1,8 @@
 /* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
-/* global PairExpiredDialog */
 
 'use strict';
```

Comment:
‘use strict' should be moved to the define function. This way it won't affect other scripts in case the scripts are merging in the build process.